fenix/app/src/main/res/values-en-rCA/strings.xml

```diff
@@ -73,7 +73,7 @@
         private mode in our new Total Private Browsing mode.
         The first parameter is the name of the app defined in app_name (for example: Firefox Nightly)
         The second parameter is the clickable link text in felt_privacy_info_card_subtitle_link_text -->
-    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private windows. %2$s</string>
+    <string name="felt_privacy_info_card_subtitle">%1$s deletes your cookies, history, and site data when you close all your private tabs. %2$s</string>
```

Comment:
We should only ever be changing the values in `fenix/app/src/main/res/values/strings.xml`. These files that are imported from Pontoon (our localization service) and should never be manually edited.

Reply:
Reset the changes to the translated files.